src/oc_span_sweeper.erl

```diff
@@ -88,6 +77,40 @@ terminate(_Reason, _State, _Data) ->
 
 %%
 
+do_gc(#data{strategy=Strategy,
+            ttl=TTL,
+            storage_size=MaxSize}) ->
+
+    StorageSize = ets:info(?SPAN_TAB, memory) * erlang:system_info({wordsize, external}),
```

Comment:
Shouldn't bother with these when `storage_size` is `infinity`.

Reply:
oh right